.appveyor.yml

```diff
@@ -0,0 +1,217 @@
+#          PYTHON_ARCH: "32"
+#          MINICONDA: C:\Miniconda34
+#          PYQT_VERSION: "None"
+#          QT_API: "pyside"
+#
+#        - NAME: "PySide2"
+#          PYTHON: "C:\\Python36-x64"
+#          PYTHON_VERSION: "3.6.3"
+#          PYTHON_ARCH: "64"
+#          MINICONDA: C:\Miniconda36-x64
+#          PYQT_VERSION: "None"
+#          QT_API: "pyside2"
+
+        # QScintilla
+
+        - NAME: "QScintilla"
+          PYTHON: "C:\\Python36-x64"
+          PYTHON_VERSION: "3.6.3"
+          PYTHON_ARCH: "64"
+          MINICONDA: C:\Miniconda36-x64
+          PYQT_VERSION: "5"
+          QSCINTILLA: "Yes"
+
+
+
+install:
+  # If there is a newer build queued for the same PR, cancel this one.
+  # The AppVeyor 'rollout builds' option is supposed to serve the same
+  # purpose but it is problematic because it tends to cancel builds pushed
+  # directly to master instead of just PR builds (or the converse).
+  # credits: JuliaLang developers.
+  - ps: if ($env:APPVEYOR_PULL_REQUEST_NUMBER -and $env:APPVEYOR_BUILD_NUMBER -ne ((Invoke-RestMethod `
+        https://ci.appveyor.com/api/projects/$env:APPVEYOR_ACCOUNT_NAME/$env:APPVEYOR_PROJECT_SLUG/history?recordsNumber=50).builds | `
+        Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
+          throw "There are newer queued builds for this pull request, failing early." }
+  
+  # Dump some debugging information about the machine.
+  - ECHO "Filesystem root:"
+  - ps: "ls \"C:/\""
+  
+  # - ECHO "Installed SDKs:"
+  # - ps: "ls \"C:/Program Files/Microsoft SDKs/Windows\""
+  #
+  - ECHO "Installed projects:"
+  - ps: "ls \"C:\\projects\""
+  - ps: "ls \"C:\\projects\\enaml\""
+  
+  # - ECHO "Environment Variables"
+  - set
+
+  # Prepend desired Python to the PATH of this build (this cannot be
+  # done from inside the powershell script as it would require to restart
+  # the parent CMD process).
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+
+  # Prepare Miniconda.
+  - "set PATH=%MINICONDA%;%MINICONDA%\\Scripts;%PATH%"
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+
+  # Avoid warning from conda info.
+  - conda install -q -n root _license
+  # Dump the setup for debugging.
+  - conda info -a
+
+  # Upgrade to the latest version of pip to avoid it displaying warnings
+  # about it being out of date.
+  - conda upgrade -q pip setuptools
+  # Conda on Py3.4 doesn't upgrade it. Try to push the point.
+  - python -m pip install -U pip
+  
+  # Allow the building of wheels.
+  - conda install -q wheel
+
+  # Check that we have the expected set-up.
+  - "ECHO We specified %PYTHON_VERSION% win%PYTHON_ARCH%"
+  - "python --version"
+  - "python -c \"import struct; print('Architecture is win'+str(struct.calcsize('P') * 8))\""
+  - pip --version
+  
+  # At the time of writing, the setup.py says Enaml depends on a dev version
+  # of Atom which is not on PyPI. Install straight from GitHub.
```

Comment:
Will look into it the setup.py may be out of date.

Reply:
I had a brief look when it came up. It seemed there is a dependency on change to Atom that has not yet been released. (I didn't look for details.) One solution is to do an Atom release, and then change the enaml setup.py to depend on the released version.